tests/mpi/refine_and_coarsen_fixed_number_07.cc

```diff
@@ -0,0 +1,187 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2009 - 2014 by the deal.II authors
```

Comment:
If refine_and_coarsen_fixed_number_02 was created in 2009 and last modified in 2014, the copyright is correct. However, your file is created in 2015 so you should use:
Copyright (C)  2015 by the deal.II authors